CMakeLists.txt

```diff
@@ -110,6 +110,9 @@ if(SILO_FOUND)
     DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/Cabana )
 endif()
 
+# find HDF5 (XDMF)
+Cabana_add_dependency( PACKAGE HDF5 COMPONENTS CXX )
```

Comment:
CXX should be C. We are not using  CXX bindings.

Reply:
fixed